.github/workflows/tests.yaml

```diff
@@ -21,5 +24,20 @@ jobs:
         name: maker
         extraPullNames: dapp
 
-    - name: Run tests
-      run: nix-shell --pure --argstr url ${{ secrets.ETH_RPC_URL }} --run 'dapp test -v --rpc'
+    - name: Run Dapp tests
+      run: nix-shell --pure --argstr url ${{ secrets.ETH_RPC_URL }} --run 'dapp test -v --rpc --rpc-block ${{ env.BLOCK }}'
```

Comment:
Where is env.BLOCK assigned? And why not run against highest block?

Reply:
Oh, I see it's hardcoded above. This might not be future-proof.

Reply:
Ignore me if you're in active development and using this PR to test the CI. :sweat_smile:

Reply:
my thinking here is that we would update block to be pinned to times when CI will pass.  Right now there are times when the CI will fail (i.e. after the spell is cast). This would allow us to lock in a passing state so we can trust ci better and know if it is failing there is something we need to look into

Reply:
I wonder if this is something that can be obtained programmatically. I think we're not going to remember to update this value since we don't often touch CI scripts.